src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1023,7 +1023,7 @@ public boolean cancelAuthenticationActivity(int requestId) {
 
     private Handler mHandler;
 
-    class CallbackHandler {
+    private static class CallbackHandler {
```

Comment:
Acutally, I think withn CallbackHandler we also shouldn't check if callback null anymore, because we don't allow null callbacks